test/src/test/java/hudson/model/DirectoryBrowserSupportTest.java

```diff
@@ -74,6 +76,15 @@
 import org.kohsuke.stapler.StaplerRequest;
 import org.kohsuke.stapler.StaplerResponse;
 
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.not;
```

Comment:
nit: Same here

Reply:
This can't be folded because a `*` is already contributing an `equalTo`